web/core/components/issues/custom-properties.tsx

```
@@ -138,6 +178,30 @@ export const CustomProperties: React.FC<CustomPropertiesProps> = ({
+        <div className="w-full">
+          {ddLoading ? (
+            <div className="text-sm text-custom-text-400 py-1">Loading options…</div>
+          ) : ddError ? (
+            <div className="text-red-500 text-sm mt-1">{ddError}</div>
+          ) : (
+            <select
+              value={value}
+              onChange={handleChange}
+              onBlur={handleBlur}
+              autoFocus
+              className="text-sm w-full border rounded px-2 py-1 focus:outline-none focus:ring-2 focus:ring-custom-primary-100"
```

Biome (2.1.2)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)